src/components/button/Button.jsx

```diff
@@ -0,0 +1,14 @@
+import './Button.css'
+import {ArrowRight} from "@phosphor-icons/react";
+
+function Button({label, variant, onClick, hasArrow}) {
+    return (
+        <button className={variant}
```

Comment:
Mocht je het nou extra, éxtra netjes willen doen, kun je switch statement gebruiken om de className te bepalen. Want als ik een spelfout maak in de `variant` die ik meegeef, dan krijgt de button geen styling omdat de CSS-class nergens mee matcht. Door de className op te bouwen met een switch statement (tussen regel 4 en 5) kun je ook altijd terugvallen op iets, als geen van de opties matcht.